packages/spotx/test/spotx.e2e.ts

```diff
@@ -203,18 +203,19 @@ describe('SpotX APIs', () => {
 
         it("Remove liquidity and receive 'RemoveLiquidity' event", async done => {
             const totalLiquidityBefore = await cennzxSpot.getTotalLiquidity(tradeAssetA);
-            const amountToRemove = 10;
-            expect(totalLiquidityBefore.gtn(amountToRemove)).toBeTruthy();
+            const removeLiquidity = 10;
+            expect(totalLiquidityBefore.gtn(removeLiquidity)).toBeTruthy();
+            const {coreAmount, assetAmount} = await cennzxSpot.assetToWithdraw(tradeAssetA, removeLiquidity);
             await cennzxSpot
-                .removeLiquidity(tradeAssetA, amountToRemove, 1, 1)
+                .removeLiquidity(tradeAssetA, removeLiquidity, assetAmount, coreAmount)
```

Comment:
we don't want to suggest our user to use return of assetToWithdraw() in removeLiquidity(). (because it will make tx fails when price changes slightly.
you can compare coreAmount, assetAmount with the value in RemoveLiquidity event.

Reply:
hmm..yeah